website/screens/components/contextual-menu/specs/ContextualMenuSpecsPage.tsx

```diff
@@ -70,11 +72,263 @@ const sections = [
   },
   {
     title: "Design tokens",
-    content: (
-      <DxcParagraph>
-        This component does not currently have design tokens.
-      </DxcParagraph>
-    ),
+    subSections: [
+      {
+        title: "Color",
+        content: (
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Component token</th>
+                <th>Element</th>
+                <th>Core token</th>
+                <th>Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <Code>backgroundColor</Code>
+                </td>
+                <td>Container</td>
+                <td>
+                  <Code>color-white</Code>
+                </td>
+                <td>#ffffff</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>borderColor</Code>
+                </td>
+                <td>Container</td>
+                <td>
+                  <Code>color-grey-200</Code>
+                </td>
+                <td>#e6e6e6</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>menuItemFontColor</Code>
+                </td>
+                <td>Menu item</td>
+                <td>
+                  <Code>color-grey-900</Code>
+                </td>
+                <td>#333333</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>hoverMenuItemBackgroundColor</Code>
+                </td>
+                <td>Menu item:hover</td>
+                <td>
+                  <Code>color-grey-100</Code>
+                </td>
+                <td>#f2f2f2</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>activeMenuItemBackgroundColor</Code>
+                </td>
+                <td>Menu item:active</td>
+                <td>
+                  <Code>color-grey-100</Code>
+                </td>
+                <td>#f2f2f2</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>selectedMenuItemBackgroundColor</Code>
+                </td>
+                <td>Menu item selected</td>
+                <td>
+                  <Code>color-purple-100</Code>
+                </td>
+                <td>#f2eafa</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>hoverSelectedMenuItemBackgroundColor</Code>
+                </td>
+                <td>Menu item:hover selected</td>
```

Comment:
I think that these should be `Menu item selected:state`

Reply:
I did it like in the Select, to be fair I didn't look other component pages.

![image](https://github.com/dxc-technology/halstack-react/assets/44321109/02f3c8d2-04e8-459a-bf57-27ffcbf9fb5b)

https://developer.dxc.com/halstack/next/components/select/specifications/

Reply:
Yes you are right it is the same in other places.